### ISSUE TITLE
Tasks that come out of accessibility remediation for pages - wage data, employer

### ISSUE THREAD

**Issue by jmmcnj** [18F/dol-whd-14c#346]
@ltsheu to assign tasks to Harun under this capacity/user story as they come up in accessiblity remediation with pages - wage data and employer

## Tasks
- [x] **File Upload Accessibility** (Related to User Story #197)

See /modules/components/attachmentField/attachmentFieldTemplate.html
- The Browse option needs to be tabbable and Enter Key accessible. 

- Change this:
`<label class="underlabel" for="{{::$id}}">File types accepted: PDF, Word, Excel, JPG, PNG</label>`
To this:
`<p class="underlabel" id="{{::$id}}_Description">File types accepted: PDF, Word, Excel, JPG, PNG</p>`

**Comment by ltsheu** [comment]
@davoodharun The current file upload component is not accessible by screen reader or keyboard. Please see the tasks above. Making these changes should help the accessibility.

**Comment by ltsheu** [comment]
@davoodharun On further investigation, the file upload accessibility fix needs to be done with styling. I will take over this task.

**Comment by ltsheu** [comment]
This issue resolved with PR #385.
